microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/java/JavaFileTextDocumentService.java

```diff
@@ -172,13 +177,13 @@ public CompletableFuture<Either<List<CompletionItem>, CompletionList>> completio
 							return true;
 						}, projectInfo) //
 						.forEach(item -> {
-							list.getItems().add(item);
+							finalizedList.getItems().add(item);

```

Comment:
does new CompletionList() guarantee getItems() is not null?

Reply:
Yes